tests/MultiWalletTest.php

```diff
@@ -577,4 +578,27 @@ public function testUserCashier(): void
         self::assertEquals($transfer->deposit->type, Transaction::TYPE_DEPOSIT);
         self::assertEquals($transfer->deposit->amount, 100);
     }
+
+    /**
+     * @return void
+     * @throws Throwable
+     */
+    public function testDecimalPlaces(): void
+    {
+        $slug = config('wallet.wallet.default.slug', 'default');
+
+        /**
+         * @var UserMulti $user
+         */
+        $user = UserMultiFactory::new()->create();
+        self::assertNull($user->getWallet($slug));
+
+        $wallet = $user->createWallet(['name' => 'Simple', 'slug' => $slug, 'decimal_places' => 6]);
+        self::assertNotNull($wallet);
+        self::assertNotNull($user->wallet);
+        self::assertEquals($user->wallet->id, $wallet->id);
+
+        $user->deposit(1000000000);
```

Comment:
Hi @rez1dent3 could you please change this to `$wallet->deposit(1000000000);` instead of `$user->deposit` and give it a try?

Reply:
Hey @rez1dent3 after double check the test `$user->deposit` is actually the same as `$wallet->deposit` so it's all correct.

I'm not sure why I'm getting that weird issue